script/playbackstrategy/modifiers/live/restartable.js

```diff
@@ -111,7 +111,6 @@ define(
 
           stop: function () {
             mediaPlayer.stop();
-            removeEventCallback(this, updateFakeTimer);
```

Comment:
Why has this been removed?

Reply:
Because due to a bug in the html5 player it wasn't actually doing anything, and it was breaking cehtml